src/App.tsx

```diff
@@ -1,26 +1,256 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { ErrorMessage, Filterby, Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos, updateTodo } from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { ErrorModal } from './components/ErrorModal/ErrorModal';
 
-const USER_ID = 0;
+const filter = (todos: Todo[], filterBy: Filterby) => {
+  switch (filterBy) {
+    case Filterby.ACTIVE:
+      return todos.filter(todo => !todo.completed);
+    case Filterby.COMPLETED:
+      return todos.filter(todo => todo.completed);
+    case Filterby.ALL:
+    default:
+      return todos;
+  }
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState(ErrorMessage.NONE);
+  const [filterBy, setFilterBy] = useState(Filterby.ALL);
+  const [newTitle, setNewTitle] = useState<string>('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
boolean shoud start with` is , has`